runtime/src/server/middleware/get_page_handler.ts

```diff
@@ -154,8 +154,9 @@ export function get_page_handler(
 		let params;
 
 		try {
-			const root_preloaded = manifest.root_preload
-				? manifest.root_preload.call(preload_context, {
+			const root_preload = manifest.root_comp.preload || (() => {});
```

Comment:
Either we should not make `root_preload` fallback to a no-op, or we should not have a case in `root_preloaded` for when `root_preload` is falsy.

Reply:
Where do we check that `root_preload` is falsy?

Reply:
Just below on the next line.

Reply:
I don't think we do? We just use `root_preload` directly without doing any additional check on it

Reply:
Either I'm going crazy, or the next statement is `const root_preloaded = root_preload ? root_preload.call(...) : {};`.

Reply:
Thanks. I was the one going crazy

I've updated it now